tests/unit/policy_test.py

```diff
@@ -920,12 +920,12 @@ def test_surface_policy_moves_back_to_object(self):
                 #  23   | OrientHorizontal | True        | False          | touch_object
                 #  24   | OrientHorizontal | True        | False          | touch_object
                 #  25   | OrientVertical   | True        | False          | touch_object
+                #  26   | MoveForward      | True        | False          | touch_object
```

Comment:
@vkakerbeck this is one of the effects of algorithm change. We always get a `MoveForward` to finish `touch_object` phase.